Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: cloud-cli
+
+cloud-cli:
```

Comment:
What is this for?

Reply:
To create a cli tool to run nuke commands or cloud commands in the future

Reply:
No need to have makefle here

Reply:
done